utils/build-apple-framework.sh

```diff
@@ -98,22 +99,15 @@ function create_universal_framework {
   cd ./destroot/Library/Frameworks || exit 1
 
   local platforms=("$@")
+  local args=""
 
   echo "Creating universal framework for platforms: ${platforms[*]}"
 
   for i in "${!platforms[@]}"; do
-    platforms[$i]="${platforms[$i]}/hermes.framework/hermes"
-  done
-
-  lipo -create -output "${platforms[0]}" "${platforms[@]}"
-
-  # Once all was linked into a single framework, clean destroot
-  # from unused frameworks
-  for platform in "${@:2}"; do
-    rm -r "$platform"
+    args+="-framework ${platforms[$i]}/hermes.framework "
   done
 
-  lipo -info "${platforms[0]}"
```

Comment:
Any particular reason why this has been removed? I think this is a good add-on for debugging purposes on the CI.

Reply:
Because **lipo -info** fails on xcframework with a fatal error: 
can't map input file: iphoneos (Invalid argument)

Reply:
Ok